usrsctplib/user_socket.c

```diff
@@ -2584,6 +2584,12 @@ usrsctp_set_ulpinfo(struct socket *so, void *ulp_info)
 	return (register_ulp_info(so, ulp_info));
 }
 
+int
+usrsctp_set_sb_threshold(struct socket *so, uint32_t sb_threshold)
+{
```

Comment:
Can you add
```
	if (so == NULL) {
		errno = EBADF;
		return (-1);
	}
```
as it is done for most other functions (and missing from `usrsctp_set_ulpinfo()`?